nncf/sparsity/schedulers.py

```diff
@@ -88,7 +88,7 @@ def epoch_step(self, next_epoch=None):
                 self._steps_per_epoch = self._steps_in_current_epoch
 
                 # Reset step and epoch step counters
-                next_epoch = -1
+                next_epoch = 0
```

Comment:
let's create test for this or check whether existing one fails

Reply:
The test already exists. The reason for these changes is removing this code
`
return 0 if self._current_epoch == -1 else self._current_epoch
`

Reply:
I can't find the test and how it was adapted to the new behaviour in your PR. Could you point to the source code of it?

Reply:
My PR does not change previous behavior. Everything works as before.

https://github.com/openvinotoolkit/nncf/blob/2b4a93a9ff3e3c44e26a9eee8275462204a732f7/tests/sparsity/test_common.py#L200-L202